features/steps/partcad-cli/commands/init.py

```
@@ -0,0 +1,134 @@
+@then("the package should be marked as private")
+def step_impl(context):
+    """
+    TODO: @alexanderilyin: -p flag is confusing, it only drops the 'import.pub' section
+    """
+    file_path = os.path.join(context.test_dir, "partcad.yaml")
+
+    # Check if the file exists
+    assert os.path.isfile(file_path), "File 'partcad.yaml' was not created"
+
+    # Read and parse the YAML content
+    with open(file_path, "r") as file:
```

Ruff (0.8.0)

Unnecessary open mode parameters

Remove open mode parameters

(UP015)